datarobot_provider/operators/connections.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2023 DataRobot, Inc. and its affiliates.
+#
+# All rights reserved.
+#
+# This is proprietary source code of DataRobot, Inc. and its affiliates.
+#
+# Released under the terms of DataRobot Tool and Utility Agreement.
+from typing import Any
+from typing import Dict
+from typing import Iterable
+
+from airflow.exceptions import AirflowException
+from airflow.exceptions import AirflowNotFoundException
+from airflow.models import BaseOperator
+
+from datarobot_provider.hooks.connections import JDBCDataSourceHook
+from datarobot_provider.hooks.datarobot import DataRobotHook
+
+DATAROBOT_MAX_WAIT = 3600
+DATAROBOT_AUTOPILOT_TIMEOUT = 86400
```

Comment:
It's always good to include in the variable name the unit for these types of things, my guess is:
```suggestion
DATAROBOT_MAX_WAIT_SEC = 3600
DATAROBOT_AUTOPILOT_TIMEOUT_SEC = 86400
```